app/handlers/handlers_resource.py

```diff
@@ -3,20 +3,27 @@
 import kopf
 
 from app.api import TwingateAPIClient
-from app.crds import K8sMetadata, ResourceSpec
+from app.crds import ResourceSpec
 from app.handlers.base import fail, success
 
 
 @kopf.on.create("twingateresource")
-def twingate_resource_create(body, meta, spec, memo, logger, patch, **kwargs):
-    logger.info("Got a create request: %s. Metadata: %s", spec, meta)
-    k8s_metadata = K8sMetadata(**meta)
+def twingate_resource_create(body, labels, spec, memo, logger, patch, **kwargs):
+    logger.info("Got a create request: %s. Labels: %s", spec, labels)
     resource = ResourceSpec(**spec)
     client = TwingateAPIClient(memo.twingate_settings, logger=logger)
+    labels = {
+        **{
+            tag["name"]: tag["value"]
+            for tag in memo.twingate_settings.default_resource_tags
+        },
+        **dict(labels),
+    }
```

Comment:
Currently, if there are duplicate tags. The labels on Resource CRD will take precedence and overwrite the default tag

Reply:
yes thats the expected behavior 👍